main.js

```diff
@@ -96,34 +143,38 @@ function toggleTitleVisibility() {
 // Creates a grid from the inputed column and rows
 function createGridItems() {
     removeItems();
-    let numberOfColumns = columnInput.value;
-    let numberOfRow = rowInput.value;
-    let itemWidth =  window.innerWidth/numberOfColumns;
+    const numberOfColumns = columnInput.value;
+    const numberOfRows = rowInput.value;
+    const itemWidth =  window.innerWidth / numberOfColumns;
 
-    for (i=0; i<(numberOfColumns * numberOfRow); i++) {
-        addItem()
+    let index = 0
```

Comment:
Where is index being used?

Reply:
It's not being used anymore, removed it.